molecule/default/molecule.yml

```diff
@@ -6,8 +6,8 @@ driver:
   name: docker
 
 platforms:
-  - name: "molecule-ansible-role-php-fpm-${MOLECULE_DISTRO:-centos-7}"
-    image: "inmotionhosting/molecule-images:${MOLECULE_DISTRO:-centos-7}"
+  - name: "molecule-ansible-role-php-fpm-${MOLECULE_DISTRO:-debian-9}"
+    image: "inmotionhosting/molecule-images:${MOLECULE_DISTRO:-debian-9}"
```

Comment:
You don't need to change this file. Run `export MOLECULE_DISTRO=debian-9` on your docker host to change which distro you're testing with.